docs/sources/enterprise/license-expiration.md

```diff
@@ -2,7 +2,7 @@
 title = "License Expiration"
 description = ""
 keywords = ["grafana", "licensing"]
-weight = 8
+weight = 9
```

Comment:
Mitch, please use at least two, preferably three digits when assigning weights. This probably predates you, so this is for future use. It just makes renumbering easier.

For example, if you head weights 100, 200, and 300, then you wouldn't have to renumber anything if you need to add a new topic in the middle. Just assign it 150 or something.